config/development/kustomization.yaml

```diff
@@ -30,6 +30,15 @@ patches:
     target:
       kind: Deployment
       name: manager
+  - path: webhook-patches.yaml
+    target:
+      group: admissionregistration.k8s.io
+      version: v1
+      kind: MutatingWebhookConfiguration
+      name: mutating-webhook.kyma-project.io
+    options:
+      allowNameChange: false
+      alloKindChange: false
```

Comment:
Since `allowNameChange` and `allowNameChange` are set to `false` by default https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/#name-and-kind-changes? we don't need to explicitly set them here, right?

Reply:
The defaults removed